roles/servicetelemetry/templates/manifest_alertmanager_config.j2

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
I'd like to highlight what I think might be a documentation problem with the setup I see here.

Our documentation[1] says "deliver alerts to an external system, for example, email, IRC, or other notification channel" and mentions that the prometheus operator reads this secret.

Now looking at this code, I think that if I want to use email, IRC, or any other notification system OTHER than SNMP (or null) I need to supply an alertmanager_manifest parameter to my service-telemetry object; otherwise the STO is going to overwrite whatever changes I might have made to the secret. Yet the alertmanager_manifest parameter is not mentioned in the documentation. In fact, I'd say that all of the `*_manifest` overrides are treated more as a development convenience than documented feature at the moment. Do we need to document and support some use of these overrides? It seems that in a case like this where they are the only option to get basic functionality, we should at least be mentioning them.

Am I missing anything here on the plan for cases like this?

[1]https://github.com/infrawatch/documentation/blob/master/doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-alertmanager.adoc